### PR TITLE
Feature/single folder build

### PR DESCRIPTION
**Pull request by beesaycheese**
Mac:
```
git clone https://github.com/datature/portal
chmod u+x ./setup-virtualenv.sh && ./setup-virtualenv.sh
. portal-build/.venv/bin/activate
python3 portal-build/portal.py (additional flags include --electron and --gpu)
```
Win:
```
git clone https://github.com/datature/portal
setup-virtualenv.sh
portal-build\.venv\Scripts\activate
python portal-build\portal.py  (additional flags include --electron and --gpu)
```

Additional Stuff changed:
1. Toaster logic (On browser mode, the blue toaster will stay forever until it detects the backend)
2. Set a global isConnected state
3. Disable inference buttons when there is no loaded model. On hover, a tooltip info will appear
4. OnComponentDidMount, the model component will call the get model list api until it gets a postive ans. During the wait, the model button is in skeleton mode
5. GPU feature to request for a restart

### CODE REVIEW

File: src/app/pages/index.js [hunk 1]
```diff
@@ -24,8 +24,9 @@ class Annotation extends React.Component {
 
     /* Global Setting / Styling State */
     this.state = {
-      useDarkTheme: false,
+      useDarkTheme: true,
```
Comment (virtuoso98) [inline src/app/pages/index.js]: Is the default now dark theme?

File: src/app/src/components/annotations/menu.jsx [hunk 2]
```diff
@@ -360,29 +364,40 @@ export default class AnnotationMenu extends Component {
...
+                  disabled={!this.props.isConnected || !this.props.loadedModel}
+                  icon="heat-grid"
+                  text="Bulk Analysis"
+                  label={<KeyCombo combo="B" />}
+                  className={
+                    this.props.userEditState === "Bulk Analysis"
+                      ? "bp3-active"
+                      : ""
+                  }
+                  onClick={() => this.props.callbacks.GetInference()}
```
Comment (virtuoso98) [inline src/app/src/components/annotations/menu.jsx]: Why is bulk analysis and single inference pointing towards the same function? 

Note: ignore this if this is addressed in another PR.

Comment (beesaycheese) [reply src/app/src/components/annotations/menu.jsx]: Yeap, it's in another PR that's not merged yet

Comment (virtuoso98) [reply src/app/src/components/annotations/menu.jsx]: Got it!

**Comment by virtuoso98** [comment]
approving this PR

**Review by virtuoso98 — APPROVED** [review]

File: src/app/pages/index.js (hunk 1 above)
Comment (beesaycheese) [reply src/app/pages/index.js]: Yeap

**Pull request merged**